src/CoreGraphics/CGColorSpace.cs

```diff
@@ -455,7 +455,7 @@ public byte[] GetColorTable ()
 		[Obsolete ("Starting with macos10.13 use 'CreateIDCCData' instead.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
 #endif
 #endif
-#if XAMCORE_4_0
+#if XAMCORE_4_0 || NET
```

Comment:
Just remove the `XAMCORE_4_0` symbol completely everywhere (makes it easier to grep to find what's left to do), so that only `NET` is used.